media/pages/sk/podporte-nas/page.html

```diff
@@ -0,0 +1,26 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+{% load page from poleno.pages %}
+
+
+<h1>Podporte nás</h1>
+<p>
```

Comment:
Pls daj odstavec milych slov pod nadpis a pred formalne informacie. Ak nevies zostylizovat, pls popytaj Majku.

Reply:
jj, aspon taky text ako si napisal teraz. Nech to nie je len cislo uctu a nic.